lib/components/component/ComponentCard.vue

```diff
@@ -27,6 +27,12 @@ v-flex.component(sm10 :elevation-10="isActiveComponent()" :class="{ active: isAc
       }
     },
 
+    mounted() {
+      if(this.component.spreadsheetId) {
+        this.ensureSheetIndexed(this.component.spreadsheetId)
```

Comment:
Great work tying into the Vue component lifecycle, and finding a pre-existing Vuex action that did what you needed!